test/remote/remote-scenario.ts

```diff
@@ -0,0 +1,7 @@
+// Server is started manually with the server.ts script
```

Comment:
This was just an idea of how to structure the test. We should get rid of this file.

Reply:
Removed